frontend/src/lib/Buttons.ts

```diff
@@ -249,6 +261,19 @@ export default class Buttons {
     }
   }
 
+  private _retryRun(selectedIds: string[], useCurrent: boolean,
+                    callback: (selectedIds: string[], success: boolean) => void): void {
+    this._dialogActionHandler(
+        selectedIds,
+        `Retry this run?`,
+        useCurrent,
+        id => Apis.runServiceApi.retryRun(id),
```

Comment:
I believe this can be changed to `Apis.runServiceApi.retryRun`

Reply:
done